projects/wave-core-new/src/lib/datatable/table/table.component.ts

```diff
@@ -122,8 +120,8 @@ export class DataTableComponent implements OnInit, AfterViewInit, OnDestroy, OnC
     }
 
     processVectorLayer(_layer: VectorLayer, metadata: VectorLayerMetadata, data: VectorData): void {
-        this.featureColumns = metadata.columns.keySeq().toArray();
-        this.displayedColumns = ['_____select', 'coordinates', 'start', 'end'].concat(this.featureColumns);
+        this.featureColumns = metadata.dataTypes.keySeq().toArray();
+        this.displayedColumns = ['_____select', 'coordinates', 'table__start', 'table__end'].concat(this.featureColumns);
```

Comment:
if select is prefixed with `_____`, why don't you prefix your variables (coorindates, table__start, table__end) with it as well?

Reply:
Added a prefix in commit 1b784235db7b4637d0979c92f2ed60b38b48c72c